package.json

```diff
@@ -70,7 +70,6 @@
     "npm": "^6.0.0"
   },
   "lint-staged": {
-    "*.js": "eslint --cache --fix",
```

Comment:
@mstruebing I don't think we'd actually changed any JS files since this was introduced. This linter is failing for a lot of reasons. I think we should add a ticket to get it added back in.